http-client/src/main/java/io/airlift/http/client/jetty/BufferingResponseListener.java

```diff
@@ -6,7 +6,8 @@
 import org.eclipse.jetty.http.HttpFields;
 import org.eclipse.jetty.http.HttpHeader;
 import org.eclipse.jetty.http.HttpMethod;
-import org.eclipse.jetty.util.BufferUtil;
+import org.eclipse.jetty.io.ByteBufferAccumulator;
```

Comment:
Can you provide rationale for `Acquire ByteBuffer from client pool` in commit message. 
It is not obvious from the code, as end of the day we are copying the data to new buffer anyway. So why the pull.
I assume this is about preventing from OOMs due to delayed direct byte buffer memory reclaiming, right?

Reply:
Done